### ISSUE TITLE
Update Pokemon Database

### ISSUE THREAD

**Issue by n2o** [Phaetec/pogo-cruncher#96]
New pokemon are coming and our database is not up-to-date. It would be nice if you help us keeping our database ready and filled with all pokemon needed. Currently it is stored in https://github.com/Phaetec/pogo-cruncher/blob/master/frontend/src/cruncher/data/pokemon.cljs

You can also provide them as JSON file or similar and I will convert it to our scheme. The last database was from an older IV analyzer, which is no longer supported.

**Comment by yroc78** [comment]
I have caught most of the new ones is there any way I can help?

**Comment by n2o** [comment]
Yes, we need the data according to the database I linked above. Surely someone has created a list with the new pokemon and their attributes.

**Comment by yroc78** [comment]
Do you have a way to get a hold of the mods at vegaspokemap.com ? Their tracker lists iv and attributes. I wish i knew how to code or crack api

**Comment by n2o** [comment]
That's not real "coding". We just have to add more pokemon to the database ;-)

**Comment by yroc78** [comment]
how would I enter these?
http://www.eurogamer.net/articles/2017-02-28-pokemon-go-moves-movesets-move-list-and-highest-dps-attacks

**Comment by yroc78** [comment]
it looks like a fairly complete list. 
why doesn't the cruncher run? I've updated to the latest version. and restarted docker. 
it blips "loading" then nothing happens.

**Comment by n2o** [comment]
We need the raw data of this website, which they don't seem to provide.

Can't confirm your issue. Tested it a minute ago with my Google (2FA) Account and our PTC testing account. Used the image with ID a3d352bda189 (see `docker images`), cruncher v0.3.3

**Comment by yroc78** [comment]
tried to update through docker this is what I got 
docker pull phaetec/pogo-cruncher
Using default tag: latest
latest: Pulling from phaetec/pogo-cruncher
Digest: sha256:995ce56aedff6c5f63e991e92e8a8d8ee74a0f79c3d669ad400e10c00ab958dc
Status: Image is up to date for phaetec/pogo-cruncher:latest

and my login attempt  give this with the JS readout
Started Cruncher Version 0.3.2
cruncher.js:605  [  0.493s] [om.next] transacted '[(status/niantic {:status true})], #uuid "8f55c09b-cd76-41df-ae9e-7ff2d62b98a9"
http://localhost:8888/favicon.ico Failed to load resource: the server responded with a status of 404 (File not found)
cruncher.js:605  [  6.531s] [om.next] transacted '[(app/loading {:status true})], #uuid "9509a5eb-7e0d-444a-9d43-36ee0c247812"
http://localhost:5000/api/login Failed to load resource: the server responded with a status of 500 (INTERNAL SERVER ERROR)
cruncher.js:605  [ 31.367s] [om.next] transacted '[(app/loading {:status false})], #uuid "e240dff7-8ed8-4787-b107-5e787e86317d"
cruncher.js:605  [ 31.377s] [om.next] transacted '[(update/error {:message nil})], #uuid "2d981316-d374-4a89-93bd-ebe59ba22000"
cruncher.js:952 Seems like you're not catching 'em all any more... INTERNAL SERVER ERROR

**Comment by n2o** [comment]
Your image is correct, but your container is using the old image. I am confused how you start your containers. I'd suggest to remove all old containers and create a fresh one to avoid confusion. `docker system prune` can help you to delete all containers, all images and then you can start pulling the latest version of the cruncher.
As you can see in your js console output, you are using the old version 0.3.2, but the latest version is 0.3.3 where I fixed the login problem.

**Comment by yroc78** [comment]
Had to clear my browser history then it loaded. 
Thank you again for the help.

**Comment by n2o** [comment]
CTRL + SHIFT + R ;-)

**Comment by freefri** [comment]
cool, when can we be able to use #98 in production? Can we use it already in development? `docker pull phaetec/pogo-cruncher:develop` `docker run -d -p 8888:8888 -p 5000:5000 phaetec/pogo-cruncher` (I'm totally lost with how Docker deployments are working)

**Comment by n2o** [comment]
Just merged it. Images are building. This takes a few minutes.

**Comment by n2o** [comment]
Tests passed. Now hoping, that Docker Hub will do its job. Sometimes it is a bit annoying

**Comment by freefri** [comment]
We could make a script to download also pokemon pictures. Where did you take the originals from?
Next milestone should be update pokemon moves as well. Do we know any reliable source? (mayby [this](https://thesilphroad.com/research))

**Comment by n2o** [comment]
There was a repository somewhere on GitHub, which provided the original assets from the game...

**Comment by freefri** [comment]
Don't you remember the repo?
I think, people is not so much into pogo anymore, we could download [this ones](http://assets.pokemon.com/assets/cms2/img/pokedex/full/152.png) meanwhile the original ones from pogo are not easily available. 
BTW, I've notice that pokemon without evolution display `Evolves to: not-in-database`  I guess we will need to add `:previous-evolutions [1 2]` in the [pokemon list](https://github.com/Phaetec/pogo-cruncher/blob/master/frontend/src/cruncher/data/pokemon.cljs) in order to fix this. Can you confirm it from development?

**Comment by n2o** [comment]
I don't mind which pokemon images we are going to show. We can choose whatever we want. The assets repository is not maintained any more.
I can check this later.

Btw: GitHub Issues are not designed as a Chat replacement. We can use our Discord channel (see the badge in the README) for further discussions

**Comment by n2o** [comment]
Btw: GitHub Issues are not designed as a Chat replacement. We can use our Discord channel (see the badge in the README) for further discussions

**Comment by losEspinoza** [comment]
I was wondering if you were looking for any help.  I've been looking for a project to jump back into code. I would love to help where I can.

**Comment by freefri** [comment]
This [tool](https://play.google.com/store/apps/details?id=com.poke.tommy.ivstatsplay) seems to be updated using new Pokemon images, stats and attacks. Maybe we could do some ~~reverse engineering~~ to get everything updated, fast and easy.
PD: let's join this on the [Discord](https://discordapp.com/channels/210684651549884416/210684698702249994) channel
